docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -124,29 +124,105 @@ You can attach more than one action. Clicking the *Add action* button will promp
 [float]
 [[defining-rules-actions-variables]]
 ===== Action variables
-Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass rule values at the time a condition is detected to an action. You can access the list of available variables using the "add rule variable" button. Although available variables differ by rule type, all rule types pass the following variables:
 
+Using the https://mustache.github.io/[Mustache] template syntax `{{variable name}}`, you can pass rule values to an action at the time a condition is detected. You can access the list of available variables using the "add rule variable" button.
+
+All rule types pass the following variables:
+
+`date`:: The date the rule scheduled the action, in ISO format.
+`kibanaBaseUrl`:: The configured <<server-publicBaseUrl,`server.publicBaseUrl`>>. If not configured, this will be empty.
 `rule.id`:: The ID of the rule.
 `rule.name`:: The name of the rule.
 `rule.spaceId`:: The ID of the space for the rule.
 `rule.tags`:: The list of tags applied to the rule.
-`date`:: The date the rule scheduled the action, in ISO format.
-`alert.id`:: The ID of the alert that scheduled the action.
+
+There is also a set of action variables specific to the action frequency:
+
+- <<alert-action-variables,For each alert>>
+- <<alert-summary-action-variables,Summary of alerts>>
+
+[float]
+[[alert-action-variables]]
+===== Action variables for each alert
+
+Although available variables differ by rule type, when the action frequency is
+**For each alert**, all rule types pass the following variables:
+
 `alert.actionGroup`:: The ID of the action group of the alert that scheduled the action.
-`alert.actionSubgroup`:: The action subgroup of the alert that scheduled the action.
 `alert.actionGroupName`:: The name of the action group of the alert that scheduled the action.
+`alert.actionSubgroup`:: The action subgroup of the alert that scheduled the action.
 `alert.flapping`:: A flag on the alert that indicates whether the alert status is changing repeatedly.
+`alert.id`:: The ID of the alert that scheduled the action.
 
-`kibanaBaseUrl`:: The configured <<server-publicBaseUrl, `server.publicBaseUrl`>>. If not configured, this will be empty.
+[float]
+[role="child_attributes"]
+[[alert-summary-action-variables]]
+===== Action variables for summary of alerts
+
+NOTE: This type of action frequency is not available for all rule types.
+
+When the action frequency is **Summary of alerts**, rules pass the following
+variables:
+
+`alerts.all.count`:: The count of all alerts.
+
+`alerts.all.data`::
+An array of objects for all alerts. The following object properties are examples; it is not a comprehensive list.
++
+.Properties of the alerts.all.data objects
```

Comment:
Can we add a caveat around this? These are only a subset of the properties available but they are also subject to change soon. We're hoping to have a settled list by 8.8 so if we can be a little fuzzy on the details until then, it might be better.

Reply:
Yup, in cases like that we usually just put a preview/experimental tag on them. I can add that here.

Reply:
Done in 9063cca114a1fb8d71a47a113a6c92623e8f0f50